lib/codemod/package.json

```diff
@@ -41,6 +41,9 @@
     "recast": "^0.16.1",
     "regenerator-runtime": "^0.13.3"
   },
+  "peerDependencies": {
```

Comment:
I think `jest-specific-snapshot` should be moved to `devDependencies` as it's only use for test purpose. And so `jest` should be added directly to `devDependencies` too, like in `/lib/postinstall` package